app/src/components/submissionPage/CharactersLeftDisplay.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import MentionElements from "../../types/MentionElements";
+import CharactersLeftDisplay from "./CharactersLeftDisplay";
+
+describe("CharactersLeftDisplay", () => {
+    const elem1: MentionElements = { id: "idlength10", display: "Jim" };
+    const elem2: MentionElements = {
+        id: "anna@softwire.com",
+        display: "Anna",
+    };
+    const msg1: String = "";
+    const msg2: String =
+        "Hi there @[Anna](anna@softwire.com) and @[Jim](idlength10), you have cool voices!";
+    test("characters remaining string renders", () => {
+        const component = renderer.create(
+            <CharactersLeftDisplay
+                snappedUsers={[elem1, elem2]}
+                message={msg2}
+            />
+        );
+        let tree = component.toJSON();
+        expect(tree).toMatchSnapshot();
+    });
+    test("characters remaining string renders", () => {
```

Comment:
Can you make the test names more descriptive? Having two tests with exactly the same name is a bit of a code smell.

Reply:
yup, done